bedrock/firefox/templates/firefox/new/trailhead/thanks-a.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero, picto_card with context %}
+
+{% extends "firefox/new/trailhead/base.html" %}
+
+{% block extrahead %}
+  {{ super() }}
+  {{ css_bundle('firefox_exp_thanks') }}
+{% endblock %}
+
+{% block site_header %}
```

Comment:
If you extend from `thanks.html` then you can also remove this block.

Reply:
I added it in because the design included the nav:
![image](https://user-images.githubusercontent.com/30483988/87689295-e5777d80-c73c-11ea-9712-bc11cc23eedb.png)

Reply:
What I mean is, the current /thanks page already has the nav - so if you extend that template then there's no need to re-include it again.

Reply:
ohh whoops, removing now.